platform-api/src/routers/platform.py

```
@@ -0,0 +1,105 @@
+            checks[name] = f"error: {str(e)}"
+
+    async def check_sanity():
+        pid = settings.env_vars.get("sanity_project_id")
+        dataset = (settings.env_vars.get("sanity_dataset_capstone") or settings.env_vars.get("sanity_dataset_rwc"))
+        if not pid or dataset: 
+            raise ValueError("No Sanity Project ID or Dataset")
+        async with get_client(timeout=4.0) as client:
+            resp = await client.get(f"https://{pid}.api.sanity.io/v2024-01-01/data/query/{dataset}?query=*[_type=='page'][0]")
+            resp.raise_for_status()
+
+    async def check_discord():
```

Ruff (0.15.12)

[warning] Missing return type annotation for private function `check_discord`

Add return type annotation: `None`

(ANN202)